features/xmtp/xmtp-client/xmtp-client.ts

```
@@ -234,33 +249,26 @@ export async function logoutXmtpClient(args: { inboxId: IXmtpInboxId; deleteData
-        error,
-        additionalMessage: "Failed to get XMTP environment",
-      }),
-    )
-  }
+  const localUrl = `http://${hostIp}:5555` as IXmtpEnv
 
-  xmtpLogger.debug(`Using XMTP environment: ${xmtpEnv}`)
+  xmtpLogger.debug(`Using XMTP custom local url: ${localUrl}`)
 
-  return xmtpEnv
+  return localUrl
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)